src/Resources/public/Admin.js

```diff
@@ -322,7 +322,25 @@ var Admin = {
         });
 
         jQuery('.sonata-filter-form', subject).on('submit', function () {
-            jQuery(this).find('[sonata-filter="true"]:hidden :input').val('');
+            var $form = jQuery(this);
+            $form.find('[sonata-filter="true"]:hidden :input').val('');
+
+            var defaultValues = $.param({'filter': JSON.parse(this.dataset.defaultValues)}).split('&'),
```

Comment:
Should we check the existence of this this.dataset.defaultValues? someone might have extended the template and I guess that would fail.

Reply:
I agree we have to check for it

Reply:
Added